lib/cli/commands/versions.js

```diff
@@ -34,7 +34,7 @@ versions.handler = async function() {
 		})
 	)).join("\n");
 
-	console.log(`\n${output}\n`);
+	process.stderr.write(`\n${output}\n\n`);
```

Comment:
This should remain as stdout to make the output parsable in scripts

Reply:
Done